cmd/nic-feature-discovery/app/app.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+property and proprietary rights in and to this material, related
+documentation and any modifications thereto. Any use, reproduction,
+disclosure or distribution of this material and related documentation
+without an express license agreement from NVIDIA CORPORATION or
+its affiliates is strictly prohibited.
+
+SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES
+SPDX-License-Identifier: LicenseRef-NvidiaProprietary
```

Comment:
Should we use this license for the project?

Reply:
good catch ! this one needs to be apache 2.0 leme fix